docs/source/installation.rst

```diff
@@ -87,7 +97,14 @@ Building from Source
 
     To compile the Python bindings:
 
+    .. note::
+
+        The ``Conda`` package manager is used here to easily install the several dependencies needed to build the ``Autodock-Vina`` python bindings (see above how to create a dedicated environment).
+
     .. code-block:: bash
 
+        $ conda activate vina
         $ cd AutoDock-Vina/build/python
-        $ python setup.py clean --all build install
+        $ conda install numpy boost-cpp swig
```

Comment:
Consider adding `-c conda-forge` explicitly in case the channel hasn't been added already.